include/profiler.rst

```diff
@@ -113,6 +113,12 @@ To use the tools, run the scripts on the command line:
   Connects to the device via RTT, plots data in real time, and saves the data.
   As command line arguments, provide a dataset name.
 
+* ``python3 merge_data.py test_p sync_event_p test_c sync_event_c test_merged``
+
+  Combines data from test_p and test_c datasets into one dataset (test_merged).
+  Provides clock drift compensation based on synchronization events: sync_event_p and sync_event_c.
+  This enables you to observe times between events for the two connected devices.
+  As command line arguments, provide the following datasets: names of events used for synchronization for a Peripheral (sync_event_p) and a Central (sync_event_c), names of the Peripheral (test_p) and the Central (test_c), and the merge result (test_merged).
```

Comment:
@b-gent 
Can names of events be marked as datasets?
Only test_p, test_c, test_merged are names of collected datasets.

Reply:
Maybe `... provide names of events used for synchronization for a Peripheral (sync_event_p) and a Central (sync_event_c), and names of datasets for: Peripheral (test_p) and the Central (test_c), and the merge result (test_merged).`
If it's ok for you I will make the change

Reply:
```suggestion
  As command line arguments, provide names of events used for synchronization for a Peripheral (sync_event_p) and a Central (sync_event_c), as well as names of datasets for: the Peripheral (test_p), the Central (test_c), and the merge result (test_merged).
```